Source/Applications/SystemCenter/Model/DataFile.cs

```diff
@@ -128,6 +135,35 @@ void ConfigureRequest(HttpRequestMessage request)
             }
         }
 
+        [HttpPost]
+        [Route("ReprocessMany")]
+        public IHttpActionResult ReprocessMany([FromBody] IEnumerable<int> ids)
+        {
+
+            if (PatchAuthCheck())
+            {
+                APIConfiguration settings = new Settings(new ConfigurationLoader(CreateDbConnection).Configure).APISettings;
+                //APIQuery query = new APIQuery(settings.Key, settings.Token, settings.Host.Split(';'));
+                APIQuery query = new APIQuery(settings.Key, settings.Token, new string[] { "http://localhost:8989" });
```

Comment:
Should this be pulling the uri from a setting instead to talk to XDA?

Reply:
good catch;
That was from debugging.